src/client/studies/MCF/IntronAnalysis.js

```diff
@@ -37,6 +37,7 @@ class IntronAnalysis extends React.Component {
   service = new MCFService()
 
   bellCurveType = 'psi'
+  heatMapType = 'psi'
```

Comment:
should I have just used one variable hear for both charts?

Reply:
No it makes sense to have 2 separate variables here.